docs/_static/style.css

```diff
@@ -1,4 +1,4 @@
-@import url("theme.css");
+@import url("styles/furo.css");
```

Comment:
/notebooks/extendxclim.html
This image should be on a white background, or use grey text (there is black, invisible text here):

![Screenshot_20240429_110720](https://github.com/Ouranosinc/xclim/assets/91751417/f33a1494-f423-4a80-9e05-1e7bada74290)

![Screenshot_20240429_110824](https://github.com/Ouranosinc/xclim/assets/91751417/f38baeda-6594-44b1-9386-219803662e91)